Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.StickyHeader</id>
+    <title>StickyHeader for Xamarin.Android</title>
+    <version>1.11</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A very simple library that allows you to stick any View as a header of a ListView, RecyclerView or ScrollView.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://components.xamarin.com/view/stickyheader</projectUrl>
```

Comment:
Use fwlinks and point back to github.  Be sure and move license file from components folder to root of the project.  `../`

Reply:
This is the link to the doc detailing it -> https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15